config/environments/development.rb

```diff
@@ -93,4 +93,6 @@
   # Raise error when a before_action's only/except options reference missing actions
   config.action_controller.raise_on_missing_callback_actions = true
   config.action_mailer.default_url_options = { host: "localhost", port: 3000 }
+  # Add configuration for skipping email notifications based on environment variable
+  config.send_email_notifications = ENV["SEND_EMAIL_NOTIFICATIONS"] == "true"
```

Comment:
Can I ask why this change was made? The notifications have a default email send constant with is per-notification (preferred). And this application will only be available in development so we'll likely want to send out the notifications regardless.

My recommendation is to remove this config. @pJeyakumar @srb- thoughts?

Reply:
@jonathanloos : I made this change looking at the way to run the seeds and not trigger all those email notifications during the process, maybe a different solution will be better.

Reply:
@hvillero did you add this because notification generation was slowing down the seeding too much?

Reply:
@srb-, sorry for the late response; yes, but also lettter_opener was creating all those files per each email notification.

Reply:
Yep, I will remove it, it will be just local.